### ISSUE TITLE
Firebase.json empty after running 'ng add @angular/fire'

### ISSUE THREAD

**Issue by johnmckay-reward** [angular/angularfire#3559]
### Version info

**Angular:**
18.1.4

**Firebase:**
13.15.0

**AngularFire:**
18.0.1

**Other (e.g. Ionic/Cordova, Node, browser, operating system):**
Node v20.11.0
macOS 14.1.1 (23B81)

### How to reproduce these conditions
* Create a new Angular project
* Use the above combination of dependencies and run *ng add @angular/fire*
* Login and select your Firebase project
* Work through the steps in the teminal, select the services to add (such as Firestore)
* firebase.json is empty
<img width="1041" alt="image" src="https://github.com/user-attachments/assets/b5a3f002-8a51-45de-9403-4170d47e4a09">

* app.config.json is updated as expected
<img width="1201" alt="image" src="https://github.com/user-attachments/assets/09638e43-38f3-4e65-b6d8-145033536534">

**Failing test unit, Stackblitz demonstrating the problem**

**Steps to set up and reproduce**

* StackBlitz is tricky to recreate as need to login to your Firebase account
* If you follow the steps above, it should reproduce the issue

**Sample data and security rules**

* n/a, I follow the steps in the README and firebase.json is still empty

### Debug output

** Errors in the JavaScript console **
* No errors in the JS console, everything looks like it has worked

** Output from `firebase.database().enableLogging(true);` **
* n/a

** Screenshots **
<img width="1041" alt="image" src="https://github.com/user-attachments/assets/cd4b6dca-5cab-4adf-99e1-ee1d1deccc28">
<img width="1201" alt="image" src="https://github.com/user-attachments/assets/34790963-8439-4cd3-86b1-368ba4008cdd">

### Expected behavior

* The firebase.json file should be populated accordingly

### Actual behavior
* We get an empty firebase.json file

**Comment by ErickCisneros** [comment]
Hi, the same issue is happening to me.

**Comment by johnmckay-reward** [comment]
Good to know not just me then :)

**Comment by johnmckay-reward** [comment]
So, it turns out **ng-deploy** is deprecated.

I was confused, as I've been using this library for a while, and added it to a new project, and didn't know that this had changed.

It was only when I downloaded the code for the library to figure out what was going on, I found this:

<img width="944" alt="image" src="https://github.com/user-attachments/assets/e7f4b702-5810-43a6-8658-e4ec8b6c170f">

Running *ng deploy* just takes you through the same setup process over and over, and it (obviously) doesn't work.

I think this needs to be made clearer, as people who have been using this library for a while will think it's broken when it's actually not.
